tests/macros/macro_lib_a.html

```diff
@@ -0,0 +1,5 @@
+{% macro inner_impl() %}hello{% endmacro inner_impl -%}
+{# https://github.com/Keats/tera/issues/532
+   Try changing self:: to lib_a:: to pass one test and fail another.
+   Try changing ::inner_impl() to ::unused() to fail in another way. #}
```

Comment:
Here we'd see the content from lib_b. The self namespace does exist, but it appears to be assigned to the first import of the two in the rendered template.